.github/ISSUE_TEMPLATE/template-graduation-application.md

```diff
@@ -0,0 +1,273 @@
+# $PROJECT Graduation Application
+
+This template provides the project with a framework to inform the TOC of their conformance to the Incubation Level Criteria. 
+
+
+Project Repo(s): $URL
+Project Site:  $URL
+Sub-Projects: $LIST
+Communication: $SLACK
+
+Project points of contacts: $NAME, $EMAIL
+
+## Graduation Criteria Summary for $PROJECT
+
+### Adoption Assertion
+
+_The project has been adopted by the following organizations in a testing and integration or production capacity:_
+* 
+
+### Criteria
+
+## Application Process Principles
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Give a presentation and engage with the domain specific TAG(s) to increase awareness**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ]  **TAG provides insight/recommendation of the project in the context of the landscape**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ]  **All project metadata and resources are [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Review and acknowledgement of expectations for [graduated](sandbox.cncf.io) projects and requirements for moving forward through the CNCF Maturity levels.**		
+   - [ ] Met during Project's application on DD-MMM-YYYY.
+
+<!-- (Project assertion goes here) --> 
+
+Completion of this due diligence document, resolution of concerns raised, and presented for public comment satisifies the Due Diligence Review criteria.
+
+- [ ] **Additional documentation as appropriate for project type, e.g.: installation documentation, end user documentation, reference implementation and/or code samples.**
+
+<!-- (Project assertion goes here) --> 
+
+## Governance and Maintainers
+
+Note: this section may be augmented by the completion of a Governance Review from TAG Contributor Strategy.
+
+### Suggested
+
+- [ ]  **Governance has continuously been iterated upon by the project as a result of their experience applying it, with the governance history demonstrating evolution of maturity alongside the project's maturity evolution.**
+
+<!-- (Project assertion goes here) --> 
+
+### Required
+
+- [ ] **Clear and discoverable project governance documentation.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Governance is up to date with actual project activities, including any meetings, elections, leadership, or approval processes.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Governance clearly documents [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/) of project direction.**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Document how the project makes decisions on leadership, contribution acceptance, requests to the CNCF, and changes to governance or project goals.**
```

Comment:
suggest changes to "makes decisions on leadership assignment". There are several other places that mention leadership roles. We may want to rewrite it to avoid confusion.